packages/webhooks/plugins/webHooked.js

```diff
@@ -0,0 +1,35 @@
+const { GQL_SCHEMA_PLUGIN } = require('@condo/keystone/plugins/utils/typing')
+const { composeNonResolveInputHook } = require('@condo/keystone/plugins/utils')
+
+const plugin = (fn) => {
```

Comment:
Custom wrapper, since default one call hook on `Test` schema, and then it predictably throw Error

Reply:
use `const validator  = require('@condo/webhooks/model-validator')` and the use `validator. CURRENT_VALIDATOR` or create a function API for that

Reply:
https://github.com/open-condo-software/condo/pull/2497/commits/86e9c28e85f95fab65325596b6d4e74903d0d7c9